pombola/core/tests/test_templatetags.py

```diff
@@ -99,3 +100,23 @@ def test_gets_dynamically_generated_value_from_key(self):
 class LazyDictLike(object):
     def __getitem__(self, key):
         return key + key
+
+
+class AddQueryParameterTest(TestCase):
```

Comment:
This looks good - I'd just add one more, which tests the behaviour that an existing value for a key would be replaced